src/lib/components/wizard/st-import-steps/StepImportReview.svelte

```
@@ -224,9 +378,55 @@
+              {/if}
+            </div>
+            <div class="flex-1 min-w-0">
+              <p class="font-medium {step.status === 'active' ? 'text-primary' : step.status === 'completed' ? 'text-muted-foreground line-through' : 'text-muted-foreground'}">
+                {step.label}
+              </p>
+              {#if step.status === 'active' && step.details}
+                <p class="text-xs text-muted-foreground mt-0.5">{step.details}</p>
+              {/if}
+            </div>
+          </div>
+        {/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)